### ISSUE TITLE
9LV200 Mk1 and 2 data

### ISSUE THREAD

**Issue by nuk92** [PygmalionOfCyprus/cmo-db-requests#1902]
### DB Selector

Both

### Baseline

3372, 3341

### Hypothetical

No

### Name

9LV200 Mk1 [Radar], 9LV200 Mk2 [Radar]

### Sensor Generation

Early 1980s

### Range (nm)

8

### Sensor Capabilities

Air Search, Surface Search, Range Information, Altitude Information, Speed Information, Heading Information

### Sensor Codes

Continuous Tracking Capability, Track While Scan (TWS), Moving Target Indicator (MTI), Pulse Doppler Radar (Full LDSD Capability), Frequency Agile

### Search/Track Frequencies

J Band (10-20 GHz) [Old X Band] 1.5-3 cm

### Lower Frequency (Hz)

15,900,000,000

### Upper Frequency (Hz)

17,100,000,000

### Scan Interval (s)

1

### Horizontal Beamwidth (deg)

1.1

### Vertical Beamwidth (deg)

7

### System Noise Level (dB)

11

### Processing Gain/Loss (dB)

40

### Peak Power (W)

65000

### Pulse Width (μs)

0.2

### PRF (Hz)

2000

### Comments

Can't find any new data on the range.
The 9LV200 has a frequency agile magnetron with a frequency agility of 670 MHz.
The Mk1 used analogue components the Mk2 switched to solid state.
Sensor generation early 70s for 9LV200 Mk1.
One 9LV200 tracker can direct 2 guns/ciws simultaneously.

### Sources

The Naval Institute Guide to World Naval Weapons Systems 1997-1998, Norman Friedman, pages 344-346

https://i.imgur.com/bPyVV3k.png

**Comment by PygmalionOfCyprus** [comment]
To confirm, this is an update, right? Not an addition? Just want to clarify work order before relabeling.

**Comment by nuk92** [comment]
Yes this is an update of the 9LV200 Mk1-3 radars of the 9LV200 sensor groups.
